packages/atomic/src/components/result-template-components-v1/atomic-result-sections/atomic-result-section-bottom-metadata.tsx

```diff
@@ -2,13 +2,12 @@ import {Element, Component} from '@stencil/core';
 import {containsVisualElement} from '../../../utils/utils';
 
 /**
- * The bottom metadata section helps the information seeker see additional
- * descriptive information about the item.
+ * This section displays additional descriptive information about the item.
  *
  * Behaviour:
  * * Has a maximum height of two lines.
- *   * It’s recommended to use `atomic-result-fields-list` to ensure fields don’t overflow from this section.
- * * Exposes the `--line-height` CSS variable so child elements can adjust to the current line height.
+ * ** We recommend that you use `atomic-result-fields-list` to ensure that the fields in this section don’t overflow.
```

Comment:
Is this double-asterix on purpose?
```suggestion
* * We recommend that you use `atomic-result-fields-list` to ensure that the fields in this section don’t overflow.
```

Reply:
The double-astrix is the nested list item for an unordered list (asciidoc).